core/trino-spi/src/main/java/io/trino/spi/connector/MaterializedViewFreshness.java

```diff
@@ -15,10 +15,6 @@
 
 import java.util.Objects;
 
-/**
- * The class is used to determine the freshness of a materialized view. The flag
- * materializedViewFresh when false, indicates that the materialized view is fresh.
```

Comment:
Just change `false` to `true` instead of deleting the comment and do this in a separate PR.

Reply:
I think it's a bit redundant anyways since there's a boolean?